### ISSUE TITLE
Tries to close on buffer change

### ISSUE THREAD

**Issue by mtfurlan** [suan/vim-instant-markdown#125]
When I switch buffers, I get a page saying
```
Oops!
Firefox doesn't allow windows to self-close.
If you want the preview window to close automatically like in other browsers, go to about:config and set dom.allow_scripts_to_close_windows to true.
```
I would expect that to show up if I closed vim or deleted the buffer, not just switched to a different buffer.

Cheers

**Comment by agusdmb** [comment]
im having the same issue, but it seems that chromium lets windows to self-close cause they do go away.

**Comment by ashwinvis** [comment]
Fixed in #147
